hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -9,7 +9,6 @@
 import typing as t
 from apig_wsgi import make_lambda_handler
 from bottle import response, error
-from enum import Enum
```

Comment:
moved this here
 https://github.com/facebook/ThreatExchange/pull/536/commits/1241af8380e82e09a1c39fd4fb118c3fabdba056#diff-ec198432ec143dc2d5426b6e26a45bc5c364ee67f28e659f7bc44f6eba3f1546L133